roles/openshift-applier/tasks/check-file-location.yml

```diff
@@ -1,40 +1,12 @@
 ---
```

Comment:
Let's remove this file and rather just use the filter in `process-file.yml` and `process-template`.